### PR TITLE
first pass FAQ

### PR DESCRIPTION
**Pull request by ychae**
First pass of an FAQ page in the docs. Text is taken from the [HTAN Data Ingress Google Doc](https://docs.google.com/document/d/1ZtY8owPTkSHlLOA45HaeA0uELADC9yPcJofzF-ITmxA/edit#heading=h.zgg5j3z44n45) with minor changes (e.g. DCC liaison table is removed and linked to the DCC Liaison page instead).

Address issues: 
https://github.com/Sage-Bionetworks/HTAN-coordination/issues/202
https://github.com/Sage-Bionetworks/HTAN-coordination/issues/225
https://github.com/Sage-Bionetworks/HTAN-coordination/issues/43

### CODE REVIEW

File: faq.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+Q: How do I upload a \<dataset type> dataset?
+
+A: Please follow the standard operating procedures for data upload: [Data Ingress SOP](https://github.com/Sage-Bionetworks/HTAN-data-ingress-documentation-draft/blob/master/DOCUMENTATION.md).
+
+<br/>
+
+Q: What format does my data need to be in for submission? E.g do I need to organize my \<dataset type> dataset in a specific file/folder hierarchy?
+
+A: The topmost level folder needs to be descriptive because it will be used in the Data Ingress Service. Your data can be in any format or hierarchy under that folder.
```
Comment (xdoan) [inline faq.md:103]: would be good to link to the docs page for this!

File: faq.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+Q: We have terabytes of imaging data. How long of an upload time should we expect? 
+
+A: This depends on your internet bandwidth, where your data is located, your internet network, and more. If you are experiencing slow uploads or data bottlenecks, your DCC liaison is available to help optimize your specific use case.
+
+<br/>
+
+Q: How should I keep track of my tissue samples and associated data? Is there an identifier schema I need to follow?
+
+A: Please follow the sample identifier standard operating procedure: [HTAN Identifiers SOP](https://bit.ly/htan_ids).
```
Comment (xdoan) [inline faq.md:133]: might want to share examples of how centers are using identifiers and "blocks" etc

**Comment by xdoan** [comment]
added some comments and might want to see if any liaisons have heard any common questions that are not part of the FAQ!

File: faq.md (hunk 1 above)
Comment (ychae) [reply faq.md]: good call, just added!

File: faq.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+layout: default
+title: Frequently Asked Questions (FAQ)
+nav_order: 5
+---
+
+## General
+
+Q: What do we need to do to copy data to the DCC?
+
+A: Contact your center liaison so we understand your specific needs, upload your release-ready data with a descriptive top-level folder name, and iteratively annotate and validate your data as new metadata standards are released by the working groups with the [Data Curator App[(https://www.synapse.org/#!Wiki:syn20681266/ENTITY)] or the API. Here is the [Data Ingress SOP](https://github.com/Sage-Bionetworks/HTAN-data-ingress-documentation-draft/blob/master/DOCUMENTATION.md) in detail.
```
Comment (clarisse-lau) [inline faq.md:11]: The 'Data Ingress SOP' is linked a few times throughout this page, but the link appears to be broken?

File: faq.md [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+
+Q: What do we need to do to copy data to the DCC?
+
+A: Contact your center liaison so we understand your specific needs, upload your release-ready data with a descriptive top-level folder name, and iteratively annotate and validate your data as new metadata standards are released by the working groups with the [Data Curator App[(https://www.synapse.org/#!Wiki:syn20681266/ENTITY)] or the API. Here is the [Data Ingress SOP](https://github.com/Sage-Bionetworks/HTAN-data-ingress-documentation-draft/blob/master/DOCUMENTATION.md) in detail.    
+
+<br/>
+
+Q: Can we use a programmatic client?
+
+A: Yes, please refer to ‘Synapse data via a programmatic client’ in the [Data Ingress SOP](https://github.com/Sage-Bionetworks/HTAN-Data-Ingress-Docs/blob/master/DOCUMENTATION.md).
```
Comment (clarisse-lau) [inline faq.md:17]: may want to link to the Synapse programmatic client page

File: faq.md (hunk 3 above)
Comment (ychae) [reply faq.md:11]: I think the links are actually meant to point to the docs 🤦🏻‍♀️ The SOP was v0, I believe -- I'm updating the links to their proper docs pages now.

File: faq.md (hunk 4 above)
Comment (ychae) [reply faq.md]: Updated

File: faq.md (hunk 2 above)
Comment (ychae) [reply faq.md]: Added in more info from the HTAN identifiers SOP and RFC but it might be a bit too wordy for an FAQ...

File: faq.md [hunk 5]
```diff
@@ -130,7 +130,35 @@ A: This depends on your internet bandwidth, where your data is located, your int
 
 Q: How should I keep track of my tissue samples and associated data? Is there an identifier schema I need to follow?
 
-A: Please follow the sample identifier standard operating procedure: [HTAN Identifiers SOP](https://bit.ly/htan_ids).
+A: Please follow the sample identifier standard operating procedure: [HTAN Identifiers SOP](https://bit.ly/htan_ids) and the [RFC HTAN Identifiers](https://docs.google.com/document/d/1K6jjWQdvtGRKUMmjC7jvt69hd3CbnSyryVWpNJVndDA/edit?pli=1#). 
+
+Each HTAN entity will be uniquely identified by an HTAN identifier.
+
+HTAN will support two formats for biological identifiers: one for research participants, and the other for biospecimens and data-files (both of which are considered derived entities).
```
Comment (milen-sage) [inline faq.md:137]: @ychae I see what you mean when saying wordy... but I think for now this makes sense. Perhaps we can think about how we can focus on examples in the FAQ that capture most common cases: e.g. we could include links to spreadsheets that have example data and identifiers

**Comment by clarisse-lau** [comment]
We had a question about whether large files should be compressed; may be a good one to add to this page... I checked in with Dave about which compressed formats the DSA can take and included his points below. Feel free to edit :)

Q: Should large imaging files be compressed?
A: Yes, please compress larger files. The DCC accepts most file compression types, though lossless compression formats such as LZW are preferred.

**Comment by ychae** [comment]
@clarisse-lau I've added in the additional question :)

**Comment by clarisse-lau** [comment]
thanks @ychae!

**Comment by ychae** [comment]
I've removed the "DRAFT" prefix in the PR -- I think it's ready for a final review and merge if it's all good to go 😁

**Review by xdoan — APPROVED** [review]

**Review by clarisse-lau — APPROVED** [review]
